server/controllers/api/videos/index.ts

```diff
@@ -184,18 +200,43 @@ async function addVideo (req: express.Request, res: express.Response) {
 
   const videoPhysicalFile = req.files['videofile'][0]
   const videoInfo: VideoCreate = req.body
+  const files = req.files
+
+  return addVideo(req, res, { videoPhysicalFile, videoInfo, files })
+}
+
+async function addVideoResumable (req: express.Request, res: express.Response) {
+  const videoPhysicalFile = res.locals.videoFile as any
+  const videoInfo: VideoCreate & { size: number} = videoPhysicalFile.metadata
+  const files = {
+    bg: {
+      path: getTmpPath(videoPhysicalFile.metadata.audioBg)
```

Comment:
@kontrollanten why did you name it `bg` and `audioBg`?

Reply:
As far as I remember the `bg` name isn't used anywhere, it was just a necessary key to be able to pass to `buildVideoThumbnailsFromReq`. But of course it'd be better to name it to `audioBg` to decrease confusion risk.